r/src/work.r

```diff
@@ -14,7 +14,7 @@ source("./list_genes.r")
 source("./cluster.r")
 
 load_data <- function() {
-    experiment_id <- Sys.getenv("EXPERIMENT_ID", unset = "5928a56c7cbff9de78974ab50765ed20")
+    experiment_id <- Sys.getenv("EXPERIMENT_ID", unset = "e52b39624588791a7889e39c617f669e")
```

Comment:
This is an experiment id that had remained to be changed from inframock's experiment change

Reply:
you sure that this needs changing? The default experiment id is `5928a56c7cbff9de78974ab50765ed20` these days (I just went to https://scp.biomage.net/ page and it got auto-completed with this experiment id)

Reply:
This is the PR that was merged before mine: https://github.com/biomage-ltd/worker/pull/76/files , it changes other parts of the worker to use the new experiment id.

Reply:
And the ui also had the autocomplete id changed so I don't know why it wasn't updated in scp. https://github.com/biomage-ltd/ui/pull/132/files

Reply:
Ah I see, so there is a new new id. Makes sense!